### PR TITLE
[ZH] Update `Beatmap`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/zh.md [hunk 1]
```diff
@@ -1,12 +1,122 @@
...
+  - beatmapset
 ---
 
 # 谱面
 
-*请勿与 [难度](/wiki/Beatmap/Difficulty) 相混淆。*
+**谱面**（有时候又叫做*谱面集*）通常包含了一组含有各种各样[击打物件](/wiki/Hit_object)的，不同星级的[难度](#difficulty)，一般只含有一首歌曲。它通常也包含可打包在 [`.osz` 后缀](/wiki/osu!_File_Formats/Osz_(file_format))文件里的其他组件：
 
-**谱面**是对音乐创建的游戏关卡。它们包含了 [击打物件](/wiki/Hit_object) 、 [timing](/wiki/Client/Beatmap_editor/Timing) 信息、难度设置，以及视觉特效。
+- 以 MP3 或 Ogg 格式存储的歌曲。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:15]:
```suggestion
- 所有难度所面向的，以 MP3 或 Ogg 格式存储的歌曲自身。
```

File: wiki/Beatmap/zh.md [hunk 2]
```diff
@@ -1,12 +1,122 @@
...
+- 用来装饰游玩区域的[背景图片 (BG)](/wiki/Beatmap/Background) 或背景视频。
+- 可在听觉层面提供反馈，甚至起到重编曲效果的[自定义音效](/wiki/Beatmapping/Hitsound)（可选）。
+- 用来讲述一个故事或衬托歌曲主体的，含有运动的图片或特殊效果的[故事板](/wiki/Storyboard)（可选）。
+- 能更改游戏元素，或改变游玩外观的[自定义皮肤](/wiki/Skinning)（可选）。
 
-谱面是 **谱面集** 的一个部分。谱面集将一些谱面集合在一起以 [提交](/wiki/Submission) 至网站并显示在 [谱面列表](https://osu.ppy.sh/beatmapsets) 中。这些在同一谱面集的谱面通常包含同一个乐曲，但 [难度](/wiki/Beatmap/Difficulty) 互不相同。
+*注意：部分谱面的视觉或听觉效果可在[视觉设置](/wiki/Client/Interface/Visual_settings)叠加层内关闭。*
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:21]: mark：需要确定overlay的常用翻译是什么。按字面意思翻译为叠加层也是可以接受的。

File: wiki/Beatmap/zh.md [hunk 3]
```diff
@@ -1,12 +1,122 @@
...
+
+## 难度
+
+*主页面：[难度](/wiki/Beatmap/Difficulty)*
+
+**难度**是以[`.osu` 后缀名](/wiki/osu!_File_Formats/Osu_(file_format))存储，包含了击打物件位置、音效、类似于 [kiai 时间](/wiki/Gameplay/Kiai_time)特殊效果的文件。它也包含一些像[谱面难度设定](/wiki/Client/Beatmap_editor/Song_Setup#difficulty)之类会影响游玩的参数。不同谱面的难度有不同的结构，有时候只能在一个[游戏模式](/wiki/Game_mode)下游玩。[谱面星级](/wiki/Beatmapping/Star_rating)系统用于可视化描述谱面需要的实力水平。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:27]:
```suggestion
**难度**是一个包含了击打物件位置、音效、诸如[kiai 时间](/wiki/Gameplay/Kiai_time)等特殊效果的文件，以[`.osu` 后缀名](/wiki/osu!_File_Formats/Osu_(file_format))来存储。它也包含一些像[难度设定](/wiki/Client/Beatmap_editor/Song_Setup#difficulty)之类会影响游玩的参数。谱面中的不同难度有不同的结构，有时候只能在一个[游戏模式](/wiki/Game_mode)下游玩。[谱面星级](/wiki/Beatmapping/Star_rating)系统用于可视化描述难度需要的实力水平。
```
**你必须修改这条。** 
 
`Difficulties of a beatmap ` 指**同一**谱面中的不同难度，谱面在定义中本身是集合。 
所有的游戏关卡均需翻译成"难度"。

File: wiki/Beatmap/zh.md [hunk 4]
```diff
@@ -1,12 +1,122 @@
...
+
+## 提交
+
+*主页面：[提交](/wiki/Submission)*
+
+谱师能把自己创作的谱面[提交](/wiki/Submission)至公共[谱面列表](https://osu.ppy.sh/beatmapsets)。同时每张谱面都属于[一个人](/wiki/Beatmap/Beatmap_host)。但谱面通常是多人合作：有些难度可能是由其他谱师[合作](/wiki/Beatmap/Beatmap_collaborations)或由其他谱师[单人制作](/wiki/Beatmap/Guest_difficulty)。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:33]:
```suggestion
谱面作者（以下简称谱师）能把自己的作品[提交](/wiki/Submission)至公共[谱面列表](https://osu.ppy.sh/beatmapsets)。同时每张谱面都属于[一个人](/wiki/Beatmap/Beatmap_host)。但谱面通常是多人合作：有些难度可能是作者与其他谱师[合作](/wiki/Beatmap/Beatmap_collaborations)或由其他谱师[单独制作](/wiki/Beatmap/Guest_difficulty)。
```
其实可以不改，只是规范一下第一次出现的名词“谱师”的定义。

File: wiki/Beatmap/zh.md [hunk 5]
```diff
@@ -1,12 +1,122 @@
...
+
+<!-- TODO: after https://github.com/ppy/osu-web/issues/5852 is resolved, this section will need an update -->
+
+提交完成后，在谱面的网页上，谱师就获得了一块额外的能用于填写元数据的区域，比如谱面介绍、语言、歌曲风格、敏感内容提示。提交后的谱面还能够修改在开始游玩时显示在游玩区域上的[标题文本](/wiki/Beatmap/Title_text)，能通过寻求[审核评估团队](/wiki/People/The_Team/Nomination_Assessment_Team)成员的帮助来修改。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:37]:
```suggestion
提交完成后，谱师就获得了一些可在谱面的网页修改的额外元数据字段，其包含谱面介绍、语言、歌曲风格、敏感内容提示等。在[审核评估团队](/wiki/People/The_Team/Nomination_Assessment_Team)成员的帮助下，谱师还能够修改在开始游玩时显示在游玩区域上的[标题文本](/wiki/Beatmap/Title_text)。
```

File: wiki/Beatmap/zh.md [hunk 6]
```diff
@@ -1,12 +1,122 @@
...
+
+### 识别
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:39]:
```suggestion
### 标识符
```

Identification词性为名词。

File: wiki/Beatmap/zh.md [hunk 7]
```diff
@@ -1,12 +1,122 @@
...
+所有提交后的谱面都会归于以下列出的一种状态，而且谱面状态有可能会变更：
+
+- [坟场 (Graveyard)](Category#graveyard)
+- [制作中/待定 (Work in Progress / Pending)](Category#work-in-progress-and-pending)
+- [过审 (Qualified)](Category#qualified)
+- [上架 (Ranked)](Category#ranked)
+- [达标 (Approved)](Category#approved)
+- [社区喜爱 (Loved)](Category#loved)
+
+有些谱面状态包含[提交规则](/wiki/Ranking_Criteria)，同时，变更为这种状态的谱面会获得[排行榜](#leaderboards)。谱面获得排行榜的主流方法是通过[谱面上架 (Ranking) 程序](/wiki/Beatmap_ranking_procedure)，另一种方法是让谱面进入[社区喜爱 (loved)](Category#loved)状态。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:60]:
```suggestion
有些谱面状态包含[提交规则](/wiki/Ranking_Criteria)，同时，变更为这种状态的谱面会获得临时或永久的[排行榜](#leaderboards)。谱面获得排行榜的主流方法是通过[谱面上架 (Ranking) 程序](/wiki/Beatmap_ranking_procedure)，另一种方法是让谱面进入[社区喜爱 (loved)](Category#loved)状态。
```

>…… and enable the beatmaps to have transient or permanent [leaderboards](https://github.com/ppy/osu-wiki/blob/master/wiki/Beatmap/en.md#leaderboards). The most popular way of achieving that ……

指出这点其实想看你有意忽略这两个词语的原因。

File: wiki/Beatmap/zh.md [hunk 8]
```diff
@@ -1,12 +1,122 @@
...
+
+## 下载谱面
+
+能使用以下几种方法来下载谱面：
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:64]:
```suggestion
能使用以下几种方法来获取谱面：
```

File: wiki/Beatmap/zh.md [hunk 9]
```diff
@@ -1,12 +1,122 @@
...
+谱面也可从非官方渠道下载。它们既没有被 osu! 官方团队检查，也不是官方团队制造的。然而，这些渠道在社区里很常用。比如：
+
+- 第三方提供的谱面镜像站，如[小夜镜像站](https://osu.sayobot.cn/home)
+- 玩家自制的谱面或谱面合集，通过 [P2P 网络](https://en.wikipedia.org/wiki/Peer-to-peer)，如 BitTorrent。
+
+## 社区
+
+### 排行榜
+
+所有模式的 osu! 玩家都会根据谱面成绩来[全球排名](/wiki/Performance_points)并[互相竞争](/wiki/Ranking)。osu! 拥有特定游戏模式的排行榜。普通玩家只能看到全球排行榜，而 [osu! 支持者](/wiki/osu!supporter)能看到其他排行榜
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:91]:
```suggestion
所有模式的 osu! 玩家都会根据谱面成绩来[全球排名](/wiki/Performance_points)并[互相竞争](/wiki/Ranking)。在 osu! 中，每个谱面均拥有独立的，不同种类的的排行榜。普通玩家只能看到全球排行榜，而 [osu! 支持者](/wiki/osu!supporter)能看到其他排行榜
```

**你必须修改这条。**  

>osu! has different types of beatmap-specific leaderboards,

这里主要强调不同谱面(其实原文甚至有误，个人认为应用difficults) 的排行榜相互独立。

File: wiki/Beatmap/zh.md [hunk 10]
```diff
@@ -1,12 +1,122 @@
...
+
+- 基于所有活跃玩家设立的全球排行榜；
+- 特定[游戏模组 (Mods)](/wiki/Game_modifier) 排行榜；
+- 国家/地区排行榜；
+- 好友排行榜。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:96]:
```suggestion
- 显示玩家与好友间排名的好友排行榜。
```

>  Friend ranking that shows a player's position on the map compared to their friends.

File: wiki/Beatmap/zh.md [hunk 11]
```diff
@@ -1,12 +1,122 @@
...
+
+## 社区
+
+### 排行榜
+
+所有模式的 osu! 玩家都会根据谱面成绩来[全球排名](/wiki/Performance_points)并[互相竞争](/wiki/Ranking)。osu! 拥有特定游戏模式的排行榜。普通玩家只能看到全球排行榜，而 [osu! 支持者](/wiki/osu!supporter)能看到其他排行榜
+
+- 基于所有活跃玩家设立的全球排行榜；
+- 特定[游戏模组 (Mods)](/wiki/Game_modifier) 排行榜；
+- 国家/地区排行榜；
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:95]:
```suggestion
- 令拥有相同国/区旗玩家竞争的国家/地区专属排行榜；
```

> Country-specific ranking made of players bearing the same flag;

File: wiki/Beatmap/zh.md [hunk 12]
```diff
@@ -1,12 +1,122 @@
...
+- 好友排行榜。
+
+官方只提供全球排行榜前 500 名玩家的成绩[回放](/wiki/Gameplay/Replay)下载，玩家可以在线观看或者保存并显示在本地排行榜上。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:98]:
```suggestion
对于每个难度，官方只提供全球排行榜前 500 名玩家成绩的[回放](/wiki/Gameplay/Replay)下载，玩家可以在线观看或者保存并显示在本地排行榜上。
```
> The top 500 scores on every difficulty of a beatmap provide [replays](https://github.com/ppy/osu-wiki/blob/master/wiki/Gameplay/Replay), which can be watched online or saved for further display in a local leaderboard.

File: wiki/Beatmap/zh.md [hunk 13]
```diff
@@ -1,12 +1,122 @@
...
+
+### 季赛、月赛与聚光灯
+
+<!-- TODO: charts, as well as Chart Assembly Team, need to be referenced here when they receive a dedicated article (issue #4685) -->
+
+<!-- TODO: would be very cool to have a separate article for osu!lazer as well (issue #4686) -->
+
+*主页面：[谱面聚光灯](/wiki/Beatmap_Spotlights)*
+
+在早期建设时，社区就已出现多种多样的能展示独特和优秀的谱面（推图）的方法。其中一种就是收集一部分已上架 (Ranked) 谱面来举办季赛或月赛，顶流玩家依靠获得的总分来排名，靠前的玩家能获得 [osu! 支持者](/wiki/osu!supporter)作为奖品。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:108]:
```suggestion
早期，社区就已出现多种方法展示独特和优秀的谱面（推图）。其中，也是最早被公式记录的一种方法就是收集一部分已上架 (Ranked) 谱面来举办月度或季度榜单，总分名列前茅的顶流玩家能获得 [osu! 支持者](/wiki/osu!supporter)作为奖品。
```

> One of the first documented approaches were monthly and seasonal charts consisting of a small subset of ranked beatmaps, where top players from leaderboards displaying total score across these maps would be awarded with [osu!supporter](https://github.com/ppy/osu-wiki/blob/master/wiki/osu!supporter).

有点翻译腔

File: wiki/Beatmap/zh.md [hunk 14]
```diff
@@ -1,12 +1,122 @@
...
+
+季赛、月赛系统逐渐演变为[谱面聚光灯](/wiki/Beatmap_Spotlights)计划。虽然它们的性质差不多，但在新版客户端 (lazer) 里显示为游玩列表 (playlist)。
+
+### 作图比赛
+
+*主页面：[竞赛](/wiki/Contests)*
+
+osu! 社区经常举办不同的作图比赛来促进谱面创新和奖励优秀的谱师。作图比赛的规模通常可以小到仅由本地社区举办，或者大到全球参与，奖励极其丰厚。获胜的玩家通常能获得 [osu! 支持者](/wiki/osu!supporter)标签和独特的个人界面徽章。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:116]:
```suggestion
osu! 社区经常举办不同的作图比赛来促进谱面创新和奖励优秀的谱师。这些作图比赛既有小到仅由本地社区举办的；也有全球参与，奖励极其丰厚的。优胜者通常能获得 [osu! 支持者](/wiki/osu!supporter)标签和独特的个人界面徽章。
```

File: wiki/Beatmap/zh.md [hunk 15]
```diff
@@ -1,12 +1,122 @@
...
+
+### 精选艺术家
+
+*主页面：[精选艺术家](/wiki/Featured_Artists)*
+
+精选艺术家是 osu! 团队专注于为 osu! 提供，获得授权，可供作图的歌曲而推进的社区项目。[精选艺术家列表](https://osu.ppy.sh/beatmaps/artists)列出了所有参与的艺术家，并提供了预先准备好的谱面模板，可供作图使用。
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:122]:
```suggestion
精选艺术家是由 [osu! 团队](/wiki/People/The_Team)负责的社区项目。其专注于为 osu! 获取具有版权的歌曲以使其用于作图。[精选艺术家列表](https://osu.ppy.sh/beatmaps/artists)列出了所有涉及的艺术家，并提供预先准备好的谱面模板以便作图使用。
```

> Featured Artists is a community program by the [osu! team](https://github.com/ppy/osu-wiki/blob/master/wiki/People/The_Team) that focuses on licensing music by various artists for osu! and making it available for mapping. The website's [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) showcases each involved artist and provides pre-timed templates for mapping.

原翻译感觉读起来有些碎碎的，主语不太清楚。

**Review by RushFTK — CHANGES_REQUESTED** [review]
two of comments are fixing incorrect translation , others are just opinions.

File: wiki/Beatmap/zh.md [hunk 16]
```diff
@@ -1,12 +1,122 @@
 ---
-outdated: true
-outdated_since: f4d914b0b826e19e50caeea90f58557065e9e0b7
+tags:
+  - 谱面
```
Comment (RushFTK) [inline wiki/Beatmap/zh.md:3]: 你再难度页面的翻译中里加了英文标签，这里也把英文标签加上吧

File: wiki/Beatmap/zh.md (hunk 1 above)
Comment (YumeMuzi) [reply wiki/Beatmap/zh.md:15]: 你这也是翻译腔啊.....而且只需要提到歌曲文件就行了

File: wiki/Beatmap/zh.md (hunk 2 above)
Comment (YumeMuzi) [reply wiki/Beatmap/zh.md:21]: 这个是可行的

File: wiki/Beatmap/zh.md (hunk 7 above)
Comment (YumeMuzi) [reply wiki/Beatmap/zh.md:60]: 我认为是原文的问题。“临时或永久的”已经包含了所有可能的状态。无需特意指出。
there is a contradiction, "transient or permanent" doesn't make any sense.

File: wiki/Beatmap/zh.md (hunk 16 above)
Comment (YumeMuzi) [reply wiki/Beatmap/zh.md:3]: 这不是有吗
*mapset beatmapset* isn't here?

File: wiki/Beatmap/zh.md (hunk 13 above)
Comment (YumeMuzi) [reply wiki/Beatmap/zh.md:108]: 被公式记录的一种方法 是什么玩意

File: wiki/Beatmap/zh.md (hunk 2 above)
Comment (RushFTK) [reply wiki/Beatmap/zh.md:21]: ok

File: wiki/Beatmap/zh.md (hunk 16 above)
Comment (RushFTK) [reply wiki/Beatmap/zh.md:3]: 我瞎了

File: wiki/Beatmap/zh.md (hunk 1 above)
Comment (RushFTK) [reply wiki/Beatmap/zh.md:15]: hmmm，先这样吧

**Review by RushFTK — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**